assets/config/tailwind.config.js

```diff
@@ -86,6 +86,10 @@ module.exports = {
                     ...defaultTheme.fontFamily.sans,
                 ],
             },
+
+            boxShadow: {
+                "input-outline": "0 0 0 3px rgba(81, 38, 104, 0.5)"
```

Comment:
I'm up for giving this a try, though I'd use a color from the maps defined above in this page, and probably one of the lighter-to-medium grays, for portability.